pkg/networkservice/common/excludedprefixes/utils.go

```diff
@@ -50,3 +55,17 @@ func exclude(source, exclude []string) []string {
 
 	return source
 }
+
+// IsEqual check if two slices contains equal strings, no matter the order
+func IsEqual(s1, s2 []string) bool {
+	s1copy := make([]string, len(s1))
+	s2copy := make([]string, len(s2))
+
+	copy(s1copy, s1)
+	copy(s2copy, s2)
+
+	sort.Strings(s1copy)
+	sort.Strings(s2copy)
+
+	return reflect.DeepEqual(s1copy, s2copy)
+}
```

Comment:
Consider using github.com/google/go-cmp/cmp

We already have dep on it https://github.com/networkservicemesh/sdk/blob/main/go.mod#L17

```suggestion
```

Reply:
@denis-tingaikin ok, done